code/go/0chain.net/smartcontract/storagesc/config.go

```diff
@@ -108,7 +106,7 @@ type Config struct {
 	// ReadPool related configurations.
 	ReadPool *readPoolConfig `json:"readpool"`
 	// WritePool related configurations.
-	WritePool *writePoolConfig `json:"writepool"`
+	WritePool *writePoolConfig `json:"allocation_pool"`
```

Comment:
Is this pool for both write and read? if yes, we may need to rename the variable WritePool.